IntelliTect.TestTools.Selenate/IntelliTect.TestTools.Selenate.Tests/ConditionalWaitTests.cs

```diff
@@ -39,9 +184,80 @@ private bool TestReturnDelegate()
             throw new Exception();
         }
 
-        private void TestNullRefException()
+        private void ThrowNullRefException()
         {
             throw new NullReferenceException();
         }
+
+        private void Equals(int expected, int value)
+        {
+            if (value == expected)
+            {
+                return;
+            }
+            throw new EqualException(expected, value);
```

Comment:
XUnit is a little bit backward. The EqualException gets thrown when Assert.Equal fails (i.e., the values you passed in were NOT equal.) However, I'm not opposed to changing this for clarity, since really I just wanted a variety of different exceptions to use to exercise the different paths for catching (or not) exceptions once the wait starts.

Reply:
Removed the method in favor of a better method that throws a specified number of exceptions